Sources/ApolloCodegenLib/CodegenConfiguration/ApolloCodegenConfiguration+OperationManifestConfiguration.swift

```diff
@@ -0,0 +1,179 @@
+import Foundation
+
+extension ApolloCodegenConfiguration {
+  
+  public struct OperationManifestConfiguration: Codable, Equatable {
+    
+    // MARK: - Properties
+    
+    /// Configures the generation of an operation manifest JSON file for use with persisted queries
+    /// or [Automatic Persisted Queries (APQs)](https://www.apollographql.com/docs/apollo-server/performance/apq).
+    /// Defaults to `nil`.
+    public let operationManifest: OperationManifestFileOutput?
+    
+    /// How to generate the operation documents for your generated operations.
+    public let operationDocumentFormat: OperationDocumentFormat
+    
+    /// If set to `true` will generate the operation manfiest every time code generation is run. Defaults to `false`
+    public let autoGenerate: Bool
```

Comment:
Can we change the name of this to `generateManifestOnCodeGeneration` or something to that end?

Reply:
Yea that would definitely be more descriptive